### ISSUE TITLE
Recurrent lymphadenopathy

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6073]
**Preferred term label:**
Recurrent lymphadenopathy

**Synonyms**
Recurrent lymph node hyperplasia

**Definition (free text, please give PubMed ID)**
Reoccuring enlargment (swelling) of a lymph node
PMID: 26684692

**Parent term (use hpo.jax.org/app)**
Lymphadenopathy

**Comment by pnrobinson** [comment]
It is probably best to use the modifier term, otherwise we would need to make "recurrent" versions of all of the locations
https://hpo.jax.org/app/browse/term/HP:0031796
